lib/src/syn/v1/stmt/define/user.rs

```diff
@@ -35,28 +33,19 @@ pub fn user(i: &str) -> IResult<&str, DefineUserStatement> {
 		Ok((i, (name, base, opts)))
 	})(i)?;
 	// Create the base statement
-	let mut res = DefineUserStatement {
+	let mut res = DefineUserStatement::from_parsed_values(
 		name,
 		base,
-		roles: vec!["Viewer".into()], // New users get the viewer role by default
-		code: rand::thread_rng()
-			.sample_iter(&Alphanumeric)
-			.take(128)
-			.map(char::from)
-			.collect::<String>(),
-		..Default::default()
-	};
+		vec!["Viewer".into()], // New users get the viewer role by default
```

Comment:
I don't know why it was capitalized this way but this question is somewhat irrelevant to this PR since this parser needs to reproduce the AST of the original parser. If this needs to be changed it should probably be done in a different PR.